### DISCUSSION TITLE
My ComfyUI was hijack by someone who using lolMiner

### DISCUSSION THREAD

**Discussion by MorrisLu-Taipei** [comfyanonymous/ComfyUI#3451] [General]
<img width="960" alt="Clipboard01" src="https://github.com/comfyanonymous/ComfyUI/assets/22585297/cb2826d5-e0fb-4be9-b6fc-eae10aee2067">
Any suggestion?
Can addon auth to prevent someone access it through internet

**Comment by ltdrdata** [comment]
It is recommended to control access through network firewalls rather than relying solely on the inherent security of ComfyUI.

**Reply by MorrisLu-Taipei** [reply]
any hint to lock install nodes from others, firewall is not a good way to do this.

Better way is providing login / password

**Reply by ltdrdata** [reply]
https://github.com/ltdrdata/ComfyUI-Manager#security-policy

Security feature is updated.

**Comment by solitaire2015** [comment]
![image](https://github.com/comfyanonymous/ComfyUI/assets/15261283/5f195771-7d86-4e78-b779-ad66b22eab8b)
I encountered the same problem,
I'm guessing this might be caused by some custom nodes. But I'm not sure which node it is.
The custom nodes I installed:
CharacterFaceSwap           
ComfyUI-Advanced-ControlNet    
comfyui_controlnet_aux        
ComfyUI_FizzNodes            
ComfyUI-KJNodes             
ComfyUi_PromptStylers             
ComfyUI-SVD                
ComfyUI-VideoHelperSuite    
example_node.py.example  
IPAdapter-ComfyUI         
rgthree-comfy            
websocket_image_save.py
comfy-image-saver           
ComfyUI-AnimateAnyone-Evolved
ComfyUI_Custom_Nodes_AlekPet
ComfyUI-Frame-Interpolation 
ComfyUI-Manager             
ComfyUI-QualityOfLifeSuit_Omar92  
comfyui-terminal           
ComfyUI-WD14-Tagger         
facerestore_cf           
masquerade-nodes-comfyui  
sdxl_prompt_styler       
wlsh_nodes
ComfyMath                   
ComfyUI-AnimateDiff-Evolved 
ComfyUI-Custom-Scripts      
ComfyUI-Impact-Pack         
comfyui-mixlab-nodes        
comfyui-reactor-node              
ComfyUI_tinyterraNodes     
custom_animate_utils        
FreeU_Advanced           
mikey_nodes               
sdxl_prompt_styler-main
comfy_sd_krita_plugin_nodes 
comfyui-art-venture         
ComfyUI_Dragos_Nodes        
ComfyUI-Inspire-Pack        
ComfyUI_NestedNodeBuilder   
ComfyUI-SaveImgPrompt             
comfyui-tooling-nodes      
Derfuu_ComfyUI_ModdedNodes  
image_switch_node.py     
NK_switched_complete.py   
SeargeSDXL
ComfyUI-3D-Pack             
ComfyUI_Comfyroll_CustomNodes
ComfyUI_essentials           
ComfyUI_IPAdapter_plus       
comfyui-prompt-reader-node  
ComfyUI-Stable-Video-Diffusion    
ComfyUI_UltimateSDUpscale  
efficiency-nodes-comfyui    
img_expand.py                           
was-node-suite-comfyui

**Reply by ltdrdata** [reply]
https://github.com/comfyanonymous/ComfyUI/issues/3473

Remove `comfyui-terminal` and lolMiner files. 
And update ComfyUI-Manager.

And check this if your environment is remote environment.
https://github.com/ltdrdata/ComfyUI-Manager#security-policy
